nixos/modules/programs/nufetch/nufetch.nix

```diff
@@ -0,0 +1,180 @@
+{
+  pkgs,
+  lib,
+  config,
+  ...
+}:
+
+with lib;
```

Comment:
New uses of `with lib;` should be avoided as the overly wide scope makes it unclear where variables are coming from